modules/apps/29-fee/types/encoding.go

```diff
@@ -0,0 +1,26 @@
+package types
```

Comment:
This is a copy of a file that already exists. We can definitely merge them together though I'm not sure where/how. Happy to do it now/in another PR if people feel strongly